aa_stripe/models.py

```diff
@@ -299,12 +322,12 @@ def charge(self):
         stripe.api_key = stripe_settings.API_KEY
         customer = StripeCustomer.get_latest_active_customer_for_user(self.user)
         self.customer = customer
-        if customer:
+        if customer and customer.default_card:
             try:
                 stripe_charge = stripe.Charge.create(
                     amount=self.amount,
                     currency="usd",
-                    customer=customer.stripe_customer_id,
```

Comment:
are you sure we can remove customer from this call?

Reply:
Yeah, the Stripe docs say for customer: "optional, either customer or source is required"